### PR TITLE
Update features.py

### PR DESCRIPTION
**Pull request by DavidFerrerBeltran**
Added an argument on boost_equipment() to disable cube boosting

### CODE REVIEW

File: classes/features.py [hunk 1]
```diff
@@ -61,11 +61,16 @@ def merge_equipment(self):
             self.send_string("d")
 
     # why does this buff cube?
```
Comment (kujan) [inline classes/features.py]: remove comment

File: classes/features.py [hunk 2]
```diff
@@ -61,11 +61,16 @@ def merge_equipment(self):
...
+        
+        Keyword arguments
+        boost_cube -- If True (default), will boost cube after all equipment
+                      has been boosted.
+        """
         self.menu("inventory")
         for slot in coords.EQUIPMENT_SLOTS:
-            if slot == "cube":
+            if boost_cube and slot == "cube":
                 self.click(*coords.EQUIPMENT_SLOTS[slot], "right")
```
Comment (kujan) [inline classes/features.py]:
```suggestion
                self.boost_cube()
```

File: classes/features.py (hunk 1 above)
Comment (DavidFerrerBeltran) [reply classes/features.py]: done

File: classes/features.py (hunk 2 above)
Comment (kujan) [reply classes/features.py]: The Navigation class keeps track of where we are (or rather, where we _should_ be) and would return early without clicking anything in this case.

https://github.com/kujan/NGU-scripts/blob/master/classes/navigation.py#L15

```python
def menu(self, target):
    """Navigate through main menu."""
    if Navigation.current_menu == target:
            return
```

**Pull request merged**